adoc/admin-updates.adoc

```diff
@@ -172,6 +175,12 @@ That warning means that Cilium is going to regenerate all internal data on the f
 It can result in temporary connection loss for pods and services which might take few minutes.
 ====
 
+Updates the local cluster definition cluster addons base manifests. Please run:
```

Comment:
This is more of a `how` but not `why`. I would suggest to add some explanation to clarify in what situation this needs to be run.